MCprep_addon/materials/generate.py

```diff
@@ -1427,14 +1427,15 @@ def matgen_cycles_simple(
 				mat.blend_method = 'HASHED'
 			if hasattr(mat, "shadow_method"):
 				mat.shadow_method = 'HASHED'
-
-	if use_emission:
-		inputs = [inp.name for inp in principled.inputs]
-		if 'Emission Strength' in inputs:  # Later 2.9 versions only.
-			principled.inputs['Emission Strength'].default_value = 1
-		links.new(
-			nodeSaturateMix.outputs[saturateMixOut[0]],
-			principled.inputs["Emission"])
+	
+	if use_emission_nodes:
+		if use_emission:
+			inputs = [inp.name for inp in principled.inputs]
+			if 'Emission Strength' in inputs:  # Later 2.9 versions only.
```

Comment:
Can we reduce the level indent with a change like this? (viewing in git, not sure if there's a second branch of the if below).

```suggestion
	
	if use_emission_nodes and use_emission:
		inputs = [inp.name for inp in principled.inputs]
		if 'Emission Strength' in inputs:  # Later 2.9 versions only.
```